docs/user/vsphere/install.md

```diff
@@ -0,0 +1,90 @@
+IPI install.
+
+To see a guided example of a UPI install, see [Install: User-Provided Infrastructure](install_upi.md)
+
+## Example: Installer-Provided Infrastructure (IPI)
+
+The steps for performing an IPI-based install are outlined [here][cloud-install]. Following this guide you may begin at
+the Download step.
+
+### Trust Root CA
+The `create install-config`, `create cluster`, and `destroy cluster` commands access the vCenter API. The vCenter's root CA certificates must be added to the system trust before connecting to the API. The certs can be downloaded from the vCenter's homepage:
+
+![vSphere 7 certs screenshot](images/certs.png)
+
+In general, the certs are located at `<vCenter>/certs/download.zip`. Once downloaded and unzipped, the certs folder contains certs for Linux, MacOS, and Windows:
+
+```console
+$ tree certs
+certs
+├── lin
+│   ├── 108f4d17.0
+│   ├── 108f4d17.r1
+│   ├── 7e757f6a.0
+│   ├── 8e4f8471.0
+│   └── 8e4f8471.r0
+├── mac
+│   ├── 108f4d17.0
+│   ├── 108f4d17.r1
+│   ├── 7e757f6a.0
+│   ├── 8e4f8471.0
+│   └── 8e4f8471.r0
+└── win
+    ├── 108f4d17.0.crt
+    ├── 108f4d17.r1.crl
+    ├── 7e757f6a.0.crt
+    ├── 8e4f8471.0.crt
+    └── 8e4f8471.r0.crl
+
+3 directories, 15 files
+```
+
+Add the certs appropriate for your OS to your system trust. For Fedora:
+
+```console
+sudo cp certs/lin/* /etc/pki/ca-trust/source/anchors
+sudo update-ca-trust extract
+```
+### Create Configuration
+
+```console
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform vsphere
+? vCenter vcsa.vmware.devcluster.openshift.com
+? Username user@e2e.local
+? Password [? for help] ********************
+INFO Connecting to vCenter vcsa.vmware.devcluster.openshift.com 
+? Datacenter example-datacenter
+? Cluster example-cluster  
+? Default Datastore example-datastore
+? Network example-network
+? Virtual IP Address for API 123.123.12.1
+? Virtual IP Address for Ingress 123.123.12.2
+? Base Domain example.com
+? Cluster Name mycluster
+? Pull Secret [? for help] **********************************************************
+```
+
+### Create Cluster
+
+```console
+$ openshift-install create cluster
+INFO Consuming Install Config from target directory
+INFO Creating infrastructure resources...
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.mycluster.example.com:6443...
+INFO API v1.18.2 up
+INFO Waiting up to 30m0s for bootstrapping to complete...
+INFO Destroying the bootstrap resources...
+INFO Waiting up to 30m0s for the cluster at https://api.mycluster.example.com:6443 to initialize...
+INFO Waiting up to 10m0s for the openshift-console route to be created...
+INFO Install complete!
+INFO To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/home/user/auth/kubeconfig'
+INFO Access the OpenShift web-console here: https://console-openshift-console.apps.mycluster.example.com
+INFO Login to the console with user: kubeadmin, password: 5char-5char-5char-5char
+```
+
```

Comment:
i think we should include https://github.com/openshift/installer/blob/master/docs/user/gcp/install.md#running-cluster for vSphere cluster. this includes a short few paragraphs on what's created by the installer.